api/api_public_blockchain.go

```diff
@@ -150,6 +150,26 @@ func (s *PublicBlockChainAPI) GetAccount(ctx context.Context, address common.Add
 	return serAcc, state.Error()
 }
 
+// GetHeaderByNumber returns the requested canonical block header.
+func (s *PublicBlockChainAPI) GetHeaderByNumber(ctx context.Context, number rpc.BlockNumber) (*types.Header, error) {
+	header, err := s.b.HeaderByNumber(ctx, number)
+	if err != nil {
+		return nil, err
+	}
+	return header, nil
+
```

Comment:
@hyochan-brown I think it can be simplified like this. What do you think? Also, there seems to be more code similar to this.
```suggestion
	return s.b.HeaderByNumber(ctx, number)
```

Reply:
Thank you for the good review.
I reflected it!